.github/workflows/main_workflow.yml

```diff
@@ -97,6 +97,7 @@ jobs:
         wget https://github.com/protocolbuffers/protobuf/releases/download/v3.19.4/protoc-3.19.4-linux-x86_64.zip
         unzip protoc-3.19.4-linux-x86_64.zip -d protoc
         sudo mv protoc/bin/protoc /usr/local/bin/protoc
+        sudo mv protoc/include/* /usr/local/include/
```

Comment:
otherwise
```
google/protobuf/descriptor.proto: File not found.
gogoproto/gogo.proto:32:1: Import "google/protobuf/descriptor.proto" was not found or had errors.
...
```